swank/src/test/scala/org/ensime/server/protocol/swank/SwankFormatsSpec.scala

```diff
@@ -225,6 +225,11 @@ class SwankFormatsSpec extends FlatSpec with Matchers with EnsimeTestData {
       ImplicitInfoReq(Left(file1), OffsetRange(0, 123))
     )
 
+    unmarshal(
```

Comment:
can you add a marshall test as well, please? (since we only really care about creating these message in ensime-server, not consuming them)